resources/lang/pl/labels.php

```diff
@@ -173,6 +173,7 @@
         'singular' => 'Transakcja',
         'plural' => 'Transakcje',
         'add_button' => 'Dodaj transakcję',
+        'export_button' => 'Transakcja eksportowa',
```

Comment:
```suggestion
        'export_button' => 'Eksportuj transakcję',
```

This is correct polish translation.

Did you use Google Translate to translate to other languages? Do you know Russian and Ukrainian?

I think I said really clear - please add **english string** to translation files for all languages.
Why? Because translator don't waste time to checking english translation files for missing strings to translate

Reply:
yes, I am using Google Translate to translate the labels. Should I revert all of them to english?

Reply:
Thanks